confignode/src/main/java/org/apache/iotdb/confignode/manager/ProcedureManager.java

```diff
@@ -24,14 +24,14 @@
 import org.apache.iotdb.confignode.conf.ConfigNodeConf;
 import org.apache.iotdb.confignode.conf.ConfigNodeDescriptor;
 import org.apache.iotdb.confignode.persistence.ProcedureInfo;
-import org.apache.iotdb.confignode.procedure.ConfigProcedureStore;
-import org.apache.iotdb.confignode.procedure.DeleteStorageGroupProcedure;
-import org.apache.iotdb.confignode.procedure.env.ConfigNodeProcedureEnv;
 import org.apache.iotdb.confignode.rpc.thrift.TStorageGroupSchema;
 import org.apache.iotdb.procedure.Procedure;
 import org.apache.iotdb.procedure.ProcedureExecutor;
+import org.apache.iotdb.procedure.env.ConfigNodeProcedureEnv;
```

Comment:
We could move procedure package under the org.apache.iotdb.confignode package.

Reply:
Fixed